extension/storage/redisstorage/config.go

```diff
@@ -0,0 +1,18 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package redisstorage // import "github.com/open-telemetry/opentelemetry-collector-contrib/extension/storage/redisstorage"
+
+import (
+	"time"
+
+	"go.opentelemetry.io/collector/config/configopaque"
+)
+
+// Config defines configuration for file storage extension.
+type Config struct {
```

Comment:
Do we want to define a `Validate` function here, or will the default be effective enough?

Reply:
The default config is enough, we can't validate much.